Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -131,6 +152,9 @@ namespace SandboxEditor
                     }
                     eventType = MouseEvent::Up;
                 }
+                AzToolsFramework::ViewportInteraction::ViewportMouseCursorRequestBus::Event(
+                    GetViewportId(), &AzToolsFramework::ViewportInteraction::ViewportMouseCursorRequestBus::Events::SetCursorMode,
+                    AzToolsFramework::CursorInputMode::CursorModeNone);
```

Comment:
update: I think this should also be wrapped in `SandboxEditor::ManipulatorMouseWrap()` right?